dir.props

```diff
@@ -124,30 +124,32 @@
 
     <DnuSourceList Include="https:%2F%2Fwww.myget.org/F/dotnet-core/" />
     <DnuSourceList Include="https:%2F%2Fwww.myget.org/F/dotnet-coreclr/" />
+    <DnuSourceList Include="https:%2F%2Fwww.myget.org/F/roslyn-nightly/" />
     <DnuSourceList Include="https:%2F%2Fwww.myget.org/F/dotnet-corefxtestdata/" />
     <DnuSourceList Include="https:%2F%2Fwww.myget.org/F/dotnet-buildtools/" />
     <DnuSourceList Include="https:%2F%2Fwww.nuget.org/api/v2/" />
+    <DnuSourceList Include="/usr/local/share/netcorePackages/" />
```

Comment:
Yes. I'm currently publishing them to the buildtools feed. Once I'm done, I'll remove both of the added sources.